### ISSUE TITLE
Could not find required mod: origins requires {calio @ [>=1.0.1]}

### ISSUE THREAD

**Issue by Reflex410** [apace100/origins-fabric#369]
it says that when I load up origins

**Comment by Reflex410** [comment]
https://user-images.githubusercontent.com/86811734/124207415-7a40cb80-dab3-11eb-8af3-8825e1d1a026.png

**Comment by bread-tab** [comment]
I'm having the same issue. Happened on a client installation but not another installation of the same version on a server.

Log:
https://paste.ee/p/vaqXd

Edit:
I didn't have Fabric API! Silly mistake, but hey, that shows one possible cause if you get this issue. I also had to get rid of OptiFabric to get the game to start.

See also:  #363 a similar issue, which was fixed for some people at least by switching to Java 16.

More logs:
Failed because of OptiFabric: https://paste.ee/p/JlyTH
Successful launch: https://paste.ee/p/n6dRA
